pandas/tests/io/json/test_normalize.py

```diff
@@ -277,14 +310,27 @@ def test_missing_field(self, author_missing_data):
         expected = DataFrame(ex_data)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("max_level", [0, 1])
+    def test_records_path_with_nested_data(self,
```

Comment:
Does this test focus on `record_path` or `max_level`? Assume the latter from change but unclear from naming